src/pages/explore/explore.tsx

```diff
@@ -1,17 +1,57 @@
 import { useDocumentTitle } from "hooks/useDocumentTitle";
-import { Sidebar } from "components/sidebar";
+import { Sidebar } from "components/sidebar/sidebar";
 import { VideoCard } from "components/cards/videoCard";
 import { useVideo } from "contexts/videoContext/videoContext";
+import axios from "axios";
+import { showToast } from "components/toast/toast";
+import { useEffect } from "react";
+import { video } from "contexts/videoContext/videoContext.type";
 
 const Explore = (): JSX.Element => {
 	const { state, dispatch } = useVideo();
 
+	const applyFilteres = (product: video) => {
```

Comment:
Filters*

Reply:
Thanks for pointing it out, I've changed it in local, will push with new branch